src/backend/server.ts

```diff
@@ -1,37 +1,34 @@
 import {NextFunction, Request, Response} from 'express';
+import userRoutes from './routes/user';
+import express from 'express';
+import mongoose from 'mongoose';
+import dotenv from 'dotenv';
+import cors from 'cors';
+import path from 'path';
 
-const express = require("express");
-const mongoose = require("mongoose");
-require("dotenv").config();
+const app = express();
+const PORT = process.env.PORT || 5000;
```

Comment:
provide ".env_example" file for environment variables.

Reply:
Updated